lib/diagnostic.js

```diff
@@ -38,18 +51,25 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = [
+  {
+    name: 'Stinky',
+    species: 'cat',
+    breed: 'unknown',
+    vocolize: 'unknown'
+  }
+];
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function() {
-  /* Your answer here */
+  /*  */
```

Comment:
+const Pet = function(name, species, breed, vocalization) {
 -  /* Your answer here */		 +  this.name = name;
 +  this.species = species;
 +  this.breed = breed;
 +  this.vocalization = vocalization;
  };		  };